src/DateInput/helpers/validate.tsx

```diff
@@ -0,0 +1,19 @@
+const daysInMonth = (m: any, y: any) => {
+  switch (m) {
+    case 1:
```

Comment:
1 is February and 0 is January? Or this is wrong and should be 2?

Reply:
It works fine. I’ve double-tested it. `isValidDate()` has its own logic which is not dependents on default months.